client/app/actions/ManageAvailabilityActions.js

```diff
@@ -6,6 +6,9 @@ import { expandRange, fromMidnightUTCDate, toMidnightUTCDate } from '../utils/mo
 import { addFlashNotification } from './FlashNotificationActions';
 import { blockChanges, unblockChanges } from '../reducers/ManageAvailabilityReducer';
 
+// Delay to show the save button checkmark before closing the winder.
+const SAVE_FINISHED_DELAY = 2000;
```

Comment:
The checkmark animation takes 0.9 seconds and the checkmark was supposed to be visible for a second, thus 2s. I don't see a need to test any other delays?